convert_weights.py

```diff
@@ -0,0 +1,56 @@
+import torch
+from collections import OrderedDict
+import hashlib
+import argparse
+
+
+def main():
+    """Converts saved checkpoint to the expected format for detoxify.
+    """
+    checkpoint = torch.load(ARGS.checkpoint, map_location=ARGS.device)
+
+    new_state_dict = {
+        "state_dict": OrderedDict(),
+        "config": checkpoint["hyper_parameters"]["config"],
+    }
+    for k, v in checkpoint["state_dict"].items():
+        name = k[6:]  # remove `model.`
```

Comment:
Check it actually starts with `model.` here

Reply:
It does